roles/ceph-nfs/templates/ceph-nfs.service.j2

```diff
@@ -11,7 +11,7 @@ ExecStart=/usr/bin/docker run --rm --net=host \
   {% if not containerized_deployment_with_kv -%}
   -v /var/lib/ceph:/var/lib/ceph \
   -v /etc/ceph:/etc/ceph \
-  -v /var/lib/ganesha:/var/lib/ganesha \
+  -v /var/lib/nfs/ganesha:/var/lib/nfs/ganesha \
```

Comment:
so `/var/lib/ganesha` doesn't exist?

Reply:
Correct, at least on my setup there is only /var/lib/nfs/ganesha used by ganesha